### DISCUSSION TITLE
Load and show model into Payload area to create data using flow

### DISCUSSION THREAD

**Discussion by jeffsmonteiro** [directus/directus#19569] [Feature Requests]
### Summary

using flow, when deciding to create a step that inserts data into the model, when selecting the model, load in the text area (payload) the data structure of the selected model to facilitate its use.

### Basic Example

on the flow creation screen:
- Add a data creation step
- Select a collection
- Show the structure of the collection inside the Payload area

### Motivation

Entering data is a sensitive task and not having the data structure visible can considerably increase the number of errors and work.

### Detailed Design

It would not be necessary to change anything in the interface, just load the data structure inside the payload area and replace it if the user selects another collection.

### Must Have:
- A - the data structure of the selected collection displayed within the payload area

### Could Have:
- D - instead of a JSON field, input fields could be displayed for each item in the data structure.

### Drawbacks

the only reason not to do this would be possible low flow usage or as priorities

### Alternatives

One button for easy access to data structure.

### Adoption Strategy

No type of data migration would be necessary, as it is a placeholder.

**Comment by Harrish23** [comment]
how to use payload i want to update directus collection 
i have title field i want to update by using webhook flows and struggle in payload and query

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
